CONTRIBUTING.md

```diff
@@ -200,13 +202,61 @@ To keep your fork and feature branches up-to-date, it is recommended to regularl
+
+The GitHub Actions [workflow][github-docs-actions-workflows] for building and deploying the documentation is defined in [.github/workflows/main.yml](.github/workflows/main.yml).
+The workflow is run on a [GitHub-hosted virtual machine ("runner")][github-docs-hosted-runners] whenever commits are pushed to the repository (to any branch) and when a pull request is opened, updated (with new commits), or reopened.
+When run, the workflow will always attempt to build the documentation, but will only deploy the built website to GitHub Pages for commits to the `main` branch on the [official repository][acrc-hpc-community-docs-repo].
+
+For a commit pushed to a branch, the workflow will run using the documentation source files present in the pushed commit.
+For a pull request, the workflow will run using the source files as they would be if the pull request was merged ([on the `refs/pull/prNumber/merge` branch][github-docs-events-trigger-push]).
+
+Information about current and past workflow runs on the official repository can be viewed in the [Actions tab on the repository GitHub page][acrc-hpc-community-docs-actions].
+
+### Actions on forks
+When you create a [fork][github-docs-forks] of the [HPC-community-docs GitHub repository][acrc-hpc-community-docs-repo], GitHub Actions will initially be disabled on the fork.
+The fork's Actions tab on GitHub will display a message indicating that workflows are not being run on the forked repository and a button that enables workflows.
+
+Once Actions are enabled on a fork of the HPC community docs repository, the workflow will run in the context of the fork whenever commits are pushed to a branch on that fork.
+
+The [workflow](.github/workflows/main.yml) will build the documentation, but will not deploy the built website to GitHub Pages, even if commits are pushed to the `main` branch of the fork.
+This is because the deployment job in the workflow will only run on the `main` branch of the official repository [official repository][acrc-hpc-community-docs-repo].
+
+GitHub Pages is also disabled when creating a fork (see the "Pages" section in the forked repository settings).
+The official repository uses the root of the `publishing-source` branch as the [publishing source for GitHub Pages][github-docs-pages-publishing-source] rather than the [default `gh-pages` branch][github-docs-pages-default-publishing-source].
+This ensures avoids automatic activation of GitHub Pages when the official repository is forked.
+
+### Do not publish to GitHub Pages on forks!
+**Please do not deploy the built documentation to GitHub Pages on your forked version of the official repository!** This can be done by modifying the [workflow file](.github/workflows/main.yml), or activating GitHub Pages and manually committing the built HTML documentation files to the publishing source branch on the fork. Please do not do this.
+
+We ask you not to do this to avoid creating duplicates in search engine results and confusing readers.
```

Comment:
```suggestion
We ask you not to publish the version of the documentation on a fork to avoid creating duplicates in search engine results and confusing readers.
```